tests/orm/test_fields.py

```diff
@@ -27,24 +28,24 @@ def test_all_entity_fields(entity_cls, data_regression):
     )
 
 
-@pytest.mark.parametrize(
-    'group,name',
-    (
-        (group, name)
-        for group in (
-            'aiida.node',
-            'aiida.data',
-        )
-        for name in EPS.select(group=group).names
-    ),
-)
-def test_all_node_fields(group, name, data_regression):
+@pytest.fixture
+def available_entry_points():
```

Comment:
Thanks, this is much better. Maybe the name of the fixture should be more specific, since you're only returning node and data entry points. Perhaps `node_and_data_entry_points`?